### DISCUSSION TITLE
1.12.1

### DISCUSSION THREAD

**Discussion by th-ch** [pear-devs/pear-desktop#304] [General]
_Note: this release is mostly a follow up of 1.12.0 (various fixes)_

- downloader plugin: fix metadata (by @Araxeus)
- downloader plugin: unlimited playlist size + better UX (by @Araxeus)
- shortcuts plugin: fix event listerner on all keys (by @Araxeus)
- fix CSS not inserted on reload (by @Araxeus)
- improve pause behavior (by @Araxeus)
- fix App ID on windows (by @Araxeus)
- bump various dependencies (improvements, security fixes)

<hr /><em>This discussion was created from the release <a href='https://github.com/th-ch/youtube-music/releases/tag/v1.12.1'>1.12.1</a>.</em>

**Comment by Bellisario** [comment]
Does this version fixes precise volume control?

**Reply by Araxeus** [reply]
@Bellisario 
What fixes are you waiting for?

if its the volume changing when next song start - #275 fixes that and other bugs (not merged yet)

If not then please share problems you encountered so they can be fixed

**Reply by Bellisario** [reply]
Yes, the solution is #275, not others. Only to have a confirm! :smile:

**Comment by BrunoZT** [comment]
so... i need some help. i acidentaly selected hide menu and i cant find plugin folder, so i cant select plugins. pls help

**Reply by Araxeus** [reply]
@BrunoZT `Alt` key shows the menu
(Or `Esc` when in-app-menu is activated)

**Comment by Good-Ol-Been** [comment]
Excuse me, I'm not sure if I'm misunderstanding the scope of the program, but I no longer appear to be able to like or dislike songs. Am I doing something wrong, does the program not allow that, or did a YouTube side update break that functionality? Thanks in advance
Edit: I may be confusing the ability with a different utility fwiw

**Reply by Bellisario** [reply]
This is a program that streams music from Youtube Music, so, you can search for songs you like and likes them (:+1:) to create your likes library. Next songs are similar songs. If you don't like a song, you can use :-1: to dislike it. Here is a screenshot of the like-dislike functionality (find it down near the music progress bar while playing a song):
![image](https://user-images.githubusercontent.com/72039923/123509999-35fb9880-d679-11eb-8e9a-7160ce548ad8.png)

**Reply by Good-Ol-Been** [reply]
> 
> 
> This is a program that streams music from Youtube Music, so, you can search for songs you like and likes them (👍) to create your likes library. Next songs are similar songs. If you don't like a song, you can use 👎 to dislike it. Here is a screenshot of the like-dislike functionality (find it down near the music progress bar while playing a song):
> ![image](https://user-images.githubusercontent.com/72039923/123509999-35fb9880-d679-11eb-8e9a-7160ce548ad8.png)

I appreciate you trying to help, but the buttons don't seem to work for me, and instead are just highlighted if i click it 
![image](https://user-images.githubusercontent.com/1490991/123510989-b35ce400-d64c-11eb-9970-be457b72abe7.png)

**Reply by Araxeus** [reply]
@Sirbab see https://github.com/th-ch/youtube-music/issues/336#issuecomment-868834408
(try disabling `Options > Resume last song when app starts`)

**Reply by Good-Ol-Been** [reply]
> 
> 
> @Sirbab see [#336 (comment)](https://github.com/th-ch/youtube-music/issues/336#issuecomment-868834408)
> (try disabling `Options > Resume last song when app starts`)

That worked a treat, thanks! <3
